### DISCUSSION TITLE
Feature Request: Accessing files from cloud storage, customizable playback speeds, and subtitles!

### DISCUSSION THREAD

**Discussion by kenantan32** [TortugaPower/BookPlayer#852] [Ideas]
Firstly, thank you for this app. It is simple and straightforward. 

As the title says. What do you devs think? Also customizing book info (author) and album art would be a great addition too!

**Comment by GianniCarlo** [comment]
Hi @waterlemon32 !  you already can change the playback speed (it can go from 0.5x to 4.0x in 0.1 increments). Regarding cloud storage, right now as long as your cloud provider has an integration to the iOS Files app via their app, you should be able to access them and import files into BookPlayer. I'm not really sure how the subtitles feature would work? if you can provide an example it would be helpful. Regarding customizing book info, yes, that's something that is planned for a future release 🙌

**Reply by kenantan32** [reply]
Yes, but I think the quick options can be adjusted. Most people want to go up to 2.5x max. And there isn't a 2x option.

I was thinking of auto generated subtitles 😊

**Reply by GianniCarlo** [reply]
But there is a 2x quick option 🤔 (see screenshot), or where do you mean? Regarding subtitles, I guess it could be done in a separate screen for accessibility purposes, but not really sure right now of the current tools available for developers to parse subtitles from audio

![IMG_E23378842955-1](https://user-images.githubusercontent.com/14112819/201111279-466dff54-2129-492d-8f80-8035745bf6d0.jpeg)
